Java Agent documentation - Configuration (#45)
undefinedlabs/scope-docs: docs/java-configuration.md

---
id: java-configuration
title: Scope Java Agent configuration
sidebar_label: Configuration
---

> ## Send arbitrary metadata.

Reviewer: ```suggestion
## Adding agent metadata
```

You can send arbitrary metadata for every test associated with certain commit which will be shown by Scope.

> Additionally, it is possible to set a environment variable as metadata value for a certain metadata key, whose final value will be obtained in runtime during the build process.

Reviewer: ```suggestion
Additionally, it is possible to set na environment variable as metadata value for a certain key, whose final value will be evaluated at runtime.
```
